.github/workflows/release.yml

```diff
@@ -39,4 +39,36 @@ jobs:
           release_name: COMPAS ${{ steps.changelog_reader.outputs.version }}
           body: ${{ steps.changelog_reader.outputs.changes }}
           prerelease: ${{ steps.changelog_reader.outputs.status == 'prereleased' }}
-          draft: ${{ steps.changelog_reader.outputs.status == 'unreleased' }}
\ No newline at end of file
+          draft: ${{ steps.changelog_reader.outputs.status == 'unreleased' }}
+
+      - name: Set up Python 3.9
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
+
+      - uses: compas-dev/compas-actions.build@v1.0.0
+        with:
+          test_lint: true
+          test_compas: true
+      - uses: NuGet/setup-nuget@v1.0.5
+      - name: Install dependencies
+        run: |
+          choco install ironpython --version=2.7.8.1
+
+      - name: 🦗 Build grasshopper components
+        uses: compas-dev/compas-actions.ghpython_components@v1
+        with:
+          source: src/compas_fab/ghpython/components
+          target: src/compas_fab/ghpython/components
```

Comment:
this correct?

Reply:
nope, sorry, did save correctly. thanks for catching this oen
now should be good.